### DISCUSSION TITLE
Idea

### DISCUSSION THREAD

**Discussion by IDontCodee** [lukea27113/lukea27113.github.io#6] [Ideas]
Can you make a back up website that you don’t change just in case the main website stops working

**Comment by mdoryammilwalrus** [comment]
No.

**Reply by lukea27113** [reply]
if you leave one more comment you will never feel the same again.....

**Reply by lukea27113** [reply]
i dont code

**Reply by IDontCodee** [reply]
why

**Reply by IDontCodee** [reply]
what did i ever do to you dude

**Reply by IDontCodee** [reply]
Im so confused please let me know what i did

**Reply by IDontCodee** [reply]
mdoryammilwalrus
do you know what I did wrong?
Please explain

**Comment by lukea27113** [comment]
you keep commenting to much
Luke Ackerman SVMS 7th grade

**Reply by IDontCodee** [reply]
Oh okay I can stop commenting as much but I don’t get why he threatened me if I was just commenting to much
